tasks/ebpf.py

```diff
@@ -525,3 +545,117 @@ def show_top_complexity_lines(
             print(lineid)
             print(source_line_to_str(int(lineno), line, compl, compinfo_widths))
             print()
+
+
+@task
+def generate_html_report(ctx: Context, dest_folder: str | Path):
+    """Generate an HTML report with the complexity data"""
+    try:
+        from jinja2 import Environment, FileSystemLoader, select_autoescape
```

Comment:
Just a question: do you lazy import because `jinja` is an "optional" deps (as in, not in the main requirements file)?

Reply:
Exactly. At least for now, as this feature is kinda experimental and I'd prefer not to add extra dependencies to the global requirements file at this stage.